CMakeLists.txt

```diff
@@ -525,7 +525,7 @@ if(LIBUV_BUILD_TESTS)
   add_executable(uv_run_tests_a ${uv_test_sources} uv_win_longpath.manifest)
   target_compile_definitions(uv_run_tests_a PRIVATE ${uv_defines})
   target_compile_options(uv_run_tests_a PRIVATE ${uv_cflags})
-  target_link_libraries(uv_run_tests_a uv_a ${uv_test_libraries})
+  target_link_libraries(uv_run_tests_a uv_a ${uv_test_libraries} -static)
```

Comment:
The `-static` is to fix qemu link issue.

Reply:
I could live with that if it was only enabled when building for qemu, otherwise it's probably better solved by installing `libc6-<arch>-cross`. `-static` doesn't always mix well with glibc.

Note that libraries (and ld.so) get installed to `/usr/<arch>-linux-gnu/lib`. I think the `gcc-<arch>-linux-gnu` packages pick that up automatically but I'm not 100% sure.

Reply:
Okay, I will add another flag to control this.